src/hotspot/os/linux/os_linux.cpp

```diff
@@ -742,7 +742,7 @@ static void *thread_native_entry(Thread *thread) {
   OSThread* osthread = thread->osthread();
   Monitor* sync = osthread->startThread_lock();
 
-  osthread->set_thread_id(os::current_thread_id());
+  osthread->set_thread_id(checked_cast<int>(os::current_thread_id()));
```

Comment:
I don't see the change @dholmes-ora.
```suggestion
  osthread->set_thread_id(checked_cast<thread_id_t>(os::current_thread_id()));
```

Reply:
Nevermind, I was looking at outdated changes.